microsoft/testsuites/core/floppy.py

```diff
@@ -30,6 +37,12 @@ class Floppy(TestSuite):
         priority=1,
     )
     def verify_floppy_module_is_blacklisted(self, node: RemoteNode) -> None:
+        os_version = node.os.information.version
+        is_centos = isinstance(node.os, CentOs)
+        if is_centos and os_version < "7.8.0":
+            raise SkippedException(
+                "CentOS below 7.8 are not receiving fixes for this issue."
```

Comment:
1. Just double check, if 7.8 should skip or not? The `os_version < "7.8.0"` means 7.7.x are skipped, but 7.8 is not. So the message is clearer, if says "Cent OS 7.7.x and below ...".
2. "this issue" is not clear in the exception message. Please explain more.